src/eddymotion/math/utils.py

```diff
@@ -43,3 +43,31 @@ def is_positive_definite(matrix):
     except np.linalg.LinAlgError:
         # Matrix is not positive definite
         return False
+
+
+def compute_angle(v1, v2, closest_polarity=False):
+    """Compute the angle between two vectors.
+
+    Parameters
+    ----------
+    v1 : :obj:`~numpy.ndarray`
+        First vector.
+    v2 : :obj:`~numpy.ndarray`
+        Second vector.
+    closest_polarity : :obj:`bool`
+        ``True`` to compute the smallest angle between the two vectors.
```

Comment:
```suggestion
        ``True`` to reverse v2 if it points toward the opposite half-sphere.

Reply:
It's okay as long as it is clear :)

However, if the angle surpasses 90 degrees, it means that, taking v1 as reference (and pole of the hemi-sphere), v2 is not in the hemi-sphere. Therefore, you reverse v2 and then the angle will be 45 deg. Without loss of generality, of course you can take v2 and set it as the pole, and everything works the same.

Reply:
i.e., you don't define the hemispheres as positive/negative, but rather around the reference vector:

Boston is in the northern hemisphere, as is Singapore. However, if you define the hemispheres around the first vector (Boston), then Singapore is in the opposite hemisphere because their angle is more than 90 deg.

Reply:
I do not see how a hemi-sphere is defined then: v1 and v2 can be in the same "north" hemi-sphere and have a 135° angle; if we reverse v2 and take 45°, they will not be in the "north" hemi-sphere, but will be in another hemi-sphere, e.g. the "east" one... So the hemi-sphere thing here is confusing to me. I may still be mistaken.

Reply:
Sure -- let's keep it as you wrote it :)